scripts/lib/install

```diff
@@ -153,8 +167,16 @@ if [ "$mem_kb" -lt 1900000 ]; then
     exit 1
 fi
 
-# setup-apt-repo does an `apt-get update`
-"$ZULIP_PATH"/scripts/lib/setup-apt-repo
+# Do package update, e.g. do `apt-get update` on Debian
+case "$os_id" in
+    Ubuntu|Debian)
```

Comment:
`$os_id` is consistently lowercase now, so this wants `ubuntu|debian` (and three others below it).

Better yet, we can add `ID_LIKE` as `$os_id_like` and test it with
```sh
case " $os_id $os_id_like " in
    *' debian '*)
        # stuff for Ubuntu and Debian and derivatives
        ;;
    *' rhel '*)
        # stuff for RHEL and CentOS and derivatives
        ;;
esac

Reply:
Why does it have to be `" $os_id $os_id_like "` instead of just `"$os_id_like"`?

Reply:
(1) `/etc/os-release` does not include `ID` in `ID_LIKE`, and (2) the leading and trailing spaces help us correctly do whole-word matches instead of substring matches:

```
" ubuntu debian "
        ^^^^^^^^
```

Reply:
I'm guessing, that Debian itself doesn't have `$os_id_like`?

Reply:
No need to guess. Documentation:
https://www.freedesktop.org/software/systemd/man/os-release.html#ID_LIKE=

Data:
```console
$ docker run --rm debian:10 cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
$ docker run --rm ubuntu:18.04 cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.3 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.3 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

Reply:
Added in 12th commit.